src/nccl_ofi_rdma.c

```diff
@@ -5992,6 +5992,8 @@ ncclResult_t nccl_net_ofi_rdma_init(nccl_ofi_topo_t *topo,
 			nccl_net_ofi_rdma_device_t *device =
 				(nccl_net_ofi_rdma_device_t *)*base_dev;
 
+			if (!device) continue;
+
 			if (device->device_rails) {
 				release_device_ofi_resources(device);
 			}
```

Comment:
GitHub won't let me comment on the correct line, but plugin can also be freed without having been allocated.